Appraisals

```diff
@@ -1,3 +1,7 @@
 appraise 'activerecord-5' do
-  gem 'activerecord', '>= 5.0.0', '< 6.0.0'
+  gem 'activerecord', '< 6.0.0'
```

Comment:
Why did this change?

Reply:
Because things were blowing up in travis but that wasn't the cause. I left it there because bundler should behave the correct way and download active record 5